pkg/backup/item_backupper.go

```diff
@@ -432,7 +437,14 @@ func (ib *itemBackupper) takePVSnapshot(obj runtime.Unstructured, log logrus.Fie
 		log.Infof("label %q is not present on PersistentVolume, checking deprecated label...", zoneLabel)
 		pvFailureDomainZone, labelFound = pv.Labels[zoneLabelDeprecated]
 		if !labelFound {
+			var k string
 			log.Infof("label %q is not present on PersistentVolume", zoneLabelDeprecated)
+			k, pvFailureDomainZone = zoneFromPVNodeAffinity(pv, awsEbsCsiZoneKey, azureCsiZoneKey, zoneLabel)
```

Comment:
In my current cluster using ebs volumes, the nodeAffinity uses "failure-domain.beta.kubernetes.io/zone" in its `required.nodeselectorterms.matchexpressions` field. While this particular volume will be caught via the zoneLabelDeprecated *label* check, for completeness we should probably add `zoneLabelDeprecated` to the `zoneFromPVNodeAffinity` args.